Packs/GreyNoise_Community/pack_metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "GreyNoise Community",
+    "description": "GreyNoise is a threat intelligence service that collects and analyzes Internet-wide scan and attack traffic. With this integration, users can access the free, GreyNoise Community API to provide enrichment data to their workflows. The full integration code can be found here: https://github.com/demisto/content/tree/master/Packs/GreyNoise_Community",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "GreyNoise",
+    "url": "https://greynoise.io",
+    "email": "hello@greynoise.io",
```

Comment:
@bradchiappetta the other pack points to "sales@greynoise.io" which address should customers email for help/support?

Reply:
@mchasepan : they should actually probably point to support@greynoise.io.  I'll update this one shortly, and the other pack when I push the ReadMe update.